### ISSUE TITLE
Howto make PNGs for each data file

### ISSUE THREAD

**Issue by trnielsen** [McStasMcXtrace/McCode#1031]
Hi,

How do I make PNG files of all data files in a data folder?

Using a bash script one can generate PNGs for all *.dat files like this:

filenames=`ls ./*.dat`
for eachfile in $filenames
do
   mcplot-matplotlib --format png $eachfile
done

The mcplot, or any of its friends, only seems to plot all data files in one big PNG image.

E.g.:
mcplot.pl -plot -ps ISIS_SANS2d_Mantid_20200618_132443/

Will only make one big ps file.

Cheers,
Torben

- PS all data

<img width="1131" alt="plot-ps-fig" src="https://user-images.githubusercontent.com/3482458/85016675-7e30d280-b16a-11ea-8ceb-fa8e20a50e16.png">

- PNG only one file
![mcplot_10](https://user-images.githubusercontent.com/3482458/85016899-e5e71d80-b16a-11ea-9488-46329e51e158.png)

**Comment by willend** [comment]
Agreed, this is in fact the only way to generate all of these images.

I would probably do
```
cd simulation_folder
filenames=`grep filename mccode.sim | cut -f2 -d:`
for eachfile in $filenames
do
mcplot-matplotlib --format png $eachfile
done
```
(You can also do ```mcplot.pl -psc $eachfile``` in the script)

**Comment by willend** [comment]
But as a feature-request, perhaps we could make an extra input-parameter to mcplot batch-generate all plots as hardcopies, should be easy to do for mcplot-matplotlib at least... 

PyQtgraph does not really have the option to do files non-interactively and I guess we shouldn't touch mcplot.pl ... ;-)
